flowclient/flowclient/client.py

```diff
@@ -1207,8 +1207,8 @@ def joined_spatial_aggregate(
         Modal or daily location query to use to localise the metric
     metric: dict
         Metric to calculate and aggregate
-    method: {"avg", "max", "min", "median", "mode", "stddev", "variance"}, default "avg"
-        Method of aggregation one of "avg", "max", "min", "median", "mode", "stddev" or "variance".
+    method: {"avg", "max", "min", "median", "mode", "stddev", "variance", "dist"}, default "avg".
+        Method of aggregation one of "avg", "max", "min", "median", "mode", "stddev", "variance" or "dist". If the metric is qualitative it will only accept the "dist" method which yields the distribution of the metric in relative terms. All of the other methods will be rejected. On the other hand, the "dist" method will be rejected for all quantitative metrics.
```

Comment:
It wasn't quite clear to me what "qualitative" means in this context until I looked at the code/examples. This is about the distinction between continuous and categorical variables, right? Would it be useful to rephrase the docstring to make this clearer, e.g. along the following lines?
```suggestion
       Method of aggregation; one of "avg", "max", "min", "median", "mode", "stddev", "variance" or "dist". If the metric refers to a categorical variable (e.g. a subscriber handset type) it will only accept the "dist" method which yields the relative distribution of possible values. All of the other methods will be rejected. On the other hand, the "dist" method will be rejected for all continuous variables.
```